src/services/kidvids/index.jsx

```
@@ -0,0 +1,425 @@
+                onClick={isLoading ? null : handleSubmit}
+                spinner={isLoading ? <Spinner size="sm" /> : null}
+              />
+            </div>
+          </div>
+        )}
+      </Form>
+    </>
+  );
+};
+
+export const KidVidsPlayer = ({ video, isOpen, onClose, dispatch }) => {
```

ESLint

[error] 'video' is missing in props validation

(react/prop-types)